haystack/components/evaluators/answer_exact_match.py

```diff
@@ -42,18 +45,21 @@ def run(
             A list of predicted answers for each question.
         :returns:
             A dictionary with the following outputs:
-            - `result` - A number from 0.0 to 1.0 that represents the proportion of questions where any predicted
+            - `scores` - A list of 0s and 1s, where 1 means that the predicted answer matched one of the ground truth.
```

Comment:
Might make sense to provide the number of matches per question instead of a binary result.